install/kubernetes/helm/istio/values.yaml

```diff
@@ -31,11 +31,14 @@ mixer:
     enabled: true
   telemetry:
     enabled: true
-  # mixer load shedding configuration.
-  # When mixer detects that it is overloaded, it starts rejecting grpc requests.
-  loadshedding:
-    # disabled, logonly or enforce
-    mode: enforce
+    # telemetry load shedding configuration.
```

Comment:
For reference: the actual values are now in subchart. I'm moving them back here, so probably not a problem.